requirements.txt

```diff
@@ -4,7 +4,7 @@ attrs==17.4.0
 google-api-core==0.1.4
 google-auth==1.3.0
 google-cloud-pubsub==0.30.1
-gordon-dns==0.0.1.dev3
+gordon-dns==0.0.1.dev5
```

Comment:
should be `0.0.1.dev6` now

Reply:
Good catch - I'll fix it for the repo/dev environment, but it won't effect the packaging + our internal deployment.